### PR TITLE
fix(client): expose donate button on navigation bar

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by oriooctopus** [comment]
@ahmadabdolsaheb Do you have a video or screenshot showing these changes?

**Comment by ahmadabdolsaheb** [comment]
Screen Shot 2022-12-22 at 7 16 20 AM
Screen Shot 2022-12-22 at 7 16 02 AM

**Comment by ahmadabdolsaheb** [comment]
This pr would also remove the donor state from the donate button.
The donate button would also link to /donate that has a thank you message and encourages additional donations.

**Comment by oriooctopus** [comment]
I'm excited to see the effect that this is going to have. Do we have analytics in place that will be able to tell a story about how this affects donation patterns?

File: client/src/components/Header/components/universal-nav.tsx [hunk 1]
```diff
@@ -75,14 +77,25 @@ export const UniversalNav = ({
           </div>
         ) : (
           <>
+            <Media minWidth={DONATE_NAV_EXPOSED_WIDTH + 1}>
+              {' '}
+              <Link
+                sameTab={false}
+                to='/donate'
+                data-test-label='nav-donate-button'
```
Comment (oriooctopus) [inline client/src/components/Header/components/universal-nav.tsx:85]: Recently at my company we started migrating away from test specific labels such as this in favor of aria labels. Aria labels should be able to serve the same purpose in tests but they also are important for accessibility. What are your thoughts?

File: client/src/components/Header/components/nav-links.tsx [hunk 2]
```diff
@@ -310,25 +311,20 @@ export class NavLinks extends Component<NavLinksProps, NavlinkStates> {
...
-        ) : (
+        <Media maxWidth={DONATE_NAV_EXPOSED_WIDTH}>
           <li key='donate'>
             <Link
               className='nav-link'
               onKeyDown={this.handleMenuKeyDown}
               sameTab={false}
               to='/donate'
+              data-test-label='dropdown-donate-button'
+              nav-donate-button
```
Comment (oriooctopus) [inline client/src/components/Header/components/nav-links.tsx:322]: What's the reason for having to test specific selectors?

File: client/src/components/Header/components/universal-nav.tsx (hunk 1 above)
Comment (bbsmooth) [reply client/src/components/Header/components/universal-nav.tsx:85]: The accessible name for the link will be the text between the `<Link>` and `</Link>` tags, so adding an `aria-label` to this link is not only not needed for accessibility but also considered an anti-pattern.

**Review by raisedadead — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**